ui/pages/reset_password.tsx

```diff
@@ -31,7 +32,15 @@ const ResetPassword: NextPage = () => {
       kratos
         .getSettingsFlow({ id: String(flowId) })
         .then((res) => setFlow(res.data))
-        .catch(handleFlowError(router, "settings", setFlow));
+        .catch(handleFlowError("settings", setFlow))
+        .catch((err: AxiosError<string>) => {
+          if (err.response?.status === 403 && isValidUrl(err.response.data)) {
+            window.location.href = err.response.data;
+            return;
+          }
+
+          return Promise.reject(err);
+        });
```

Comment:
minor: When I try login to reset the password of a user who has enabled mfa, I momentarily see the reset_password page and then I get redirected to the authentication code page and then I log in and reset the password. My question is whether we could move this earlier so that the reset_password page does not load at all if the user needs to be redirected?

Reply:
As this request to fetch the flow happens only after the initial render we need to show something. That could be a loading screen, but I am not sure that will be a better experience -- seeing the loading screen / or a blank screen momentarily.

Reply:
Changed this to show a blank page as long as the flow is loading. So we don't have a flicker of showing the "reset password" form in between landing and the redirect.